docs/concepts/codesharing.md

```diff
@@ -129,7 +129,7 @@ A particular version of the .NET Standard Library is defined by:
   
 -  The individual types and members found in those reference assemblies.
 
-Any change in a reference assembly's API surface area causes the .NET Standard Library version to change (and its version number to increment). For example, adding a new method, such as a new `String.Contains(String, StringComparison)` overload to the [http://dotnet.github.io/api/System.String.html](http://dotnet.github.io/api/System.String.html "String") class in the System.Runtime reference assembly would cause the .NET Standard Library version to change.
+Any change in a reference assembly's API surface area causes the .NET Standard Library version to change (and its version number to increment). For example, adding a new method, such as a new `String.Contains(String, StringComparison)` overload to the [https://docs.microsoft.com/dotnet/core/api/System.String](https://docs.microsoft.com/dotnet/core/api/System.String "String") class in the System.Runtime reference assembly would cause the .NET Standard Library version to change.
```

Comment:
@rpetrusha this link text should probably be just System.String right? /cc @stevehoag